### DISCUSSION TITLE
Editor sum in another dialog

### DISCUSSION THREAD

**Discussion by mgdm187** [serenity-is/Serenity#5912] [Help]
Hi,
I am trying to figure out how to add sum of the fields in editor that is places in another dialog. I need this: https://cloud.githubusercontent.com/assets/2145218/17689102/dd2a2706-63a2-11e6-96d6-a7b7906604bc.png but I can't find the answer.
And also, where can I find the code of demo?

Thank you in advance.
BR

**Comment by edwardch** [comment]
Check out [https://github.com/serenity-is/common-features/blob/master/src/Serenity.Demo.BasicSamples/Serenity.Demo.BasicSamples/Grids/GroupingAndSummariesInGrid/GroupingAndSummariesInGrid.ts](https://github.com/serenity-is/common-features/blob/master/src/Serenity.Demo.BasicSamples/Serenity.Demo.BasicSamples/Grids/GroupingAndSummariesInGrid/GroupingAndSummariesInGrid.ts).

The demo samples have recently been moved into the [common-features](https://github.com/serenity-is/common-features) repository, and the links in the demo are still being updated to point to the correct code.

**Reply by mgdm187** [reply]
Thank you for your answer, but this didn't solve my problem. That code is for grid, and I want to have summary in editor that is places in master detail dialog. I tried this but it doesn't do anything.
